test/python/ibmq/test_ibmq_qobj.py

```diff
@@ -13,88 +13,87 @@
 
 import os
 import unittest
+import functools
 from qiskit import (ClassicalRegister, QuantumCircuit, QuantumRegister, compile)
-
 from qiskit import IBMQ, Aer
 from qiskit.qasm import pi
-from ..common import requires_qe_access, JobTestCase, slow_test
+
+from ..common import require_multiple_credentials, JobTestCase, slow_test
+
+# Timeout duration
+TIMEOUT = os.getenv("IBMQ_TIMEOUT", 10)
```

Comment:
Is `IBMQ_TIMEOUT` environment variable used elsewhere? If not and if it is specific for tests, can we change the name to `IBMQ_TESTS_TIMEOUT` or something more explicit?

Reply:
I will change it to this.